Makefile

```diff
@@ -45,14 +46,15 @@ vet:
 
 lint: golangci-lint
 	$(GOLANGCI_LINT) run
-
+lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
+	$(GOLANGCI_LINT) run --fix
 # Build the docker image
 docker-build:
-	docker build . -t ${IMG}
+	docker build . -t ${IMG}:$(VERSION)
```

Comment:
Wouldn't it be better to call `make docker-build IMG=quay.io/joelanford/helm-operator:$VERSION`? Appending `$(VERSION)` is unexpected behavior IMO.